project-base/app/src/FrontendApi/Mutation/Cart/CartMutation.php

```diff
@@ -116,13 +116,17 @@ public function addOrderItemsToCartMutation(
 
         if (!$shouldMerge) {
             $this->cartFacade->deleteCart($cart);
-            $cart = $this->cartFacade->getCartCreateIfNotExists(null, $cartUuid);
+            $cart = $this->cartFacade->getCartCreateIfNotExists($customerUser, $cartUuid);
         }
 
         $notAddedProducts = [];
         $addProductResults = [];
 
         foreach ($order->getProductItems() as $orderItem) {
+            if ($orderItem->getPromoCodeIdentifier() !== null) {
```

Comment:
This identifier is removed in #3088 (https://github.com/shopsys/shopsys/pull/3088/commits/1bc3201a47937f4a20b0626772e6d54189927d74)
We should think about a different solution